polaris-for-vscode/server/src/server.ts

```diff
@@ -40,6 +41,14 @@ let tokenGroupPatterns: TokenGroupPatterns = {
   motion: /animation/,
 };
 
+type GroupedCompletionItem = {
+  [T in GroupedTokensKey]: CompletionItem[];
+};
+
+let groupedCompletionItems: Partial<GroupedCompletionItem> = {};
```

Comment:
```suggestion
type GroupedCompletionItems = {
  [T in GroupedTokensKey]?: CompletionItem[];
};

let groupedCompletionItems: GroupedCompletionItems = {};
```
Didn't think about this during our pairing, but we can just remove the `Partial` utility type in favor of making the properties optional in the mapped type.

Note: The above PR suggestion includes a type name update to `GroupedCompletionItems` (plural) to match the variable it's assigned to (: